<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QWidget" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTR Demo</string>
  </property>
  <layout class="QHBoxLayout" name="appWindowLayout">
   <item>
    <widget class="rtrGLWidget" name="openGLWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ui_container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="modelComboBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Duck</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trefoil</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cube</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="materialComboBox">
        <item>
         <property name="text">
          <string>Toon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phong</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>274</width>
           <height>16</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>274</width>
           <height>16</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Radius</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="radius_sb"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Density</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="density_sb"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_wobble">
        <property name="text">
         <string>Wobble</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_rotate">
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>368</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>'h': show/hide UI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rtrGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>rtrglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
